Engine/src/Platform/OpenGL41/OpenGL41Check.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+namespace elv {
+void glCheckError(
+    const char* file,
+    int line,
+    const char* expression);
+}
+
+#ifdef DEBUG_MODE
```

Comment:
Not sure if PROFILE_MODE is the same as DEBUG_MODE and not sure at all where this macro comes from

Reply:
`PROFILE_MODE` macro is used only to enable/disable the internal engine profiler (`Profiler.h`)

Reply:
So there is no way currently to know if engine is compiled in debug mode or no? If no, I guess we should add a macro for it - the glChecks would probably be a pointless overhead in release/production